src/lib/urls.js

```diff
@@ -30,3 +30,39 @@ export function getCanonicalPath(path) {
   }
   return parts.join('/');
 }
+
+export function copyLinkToClipboard() {
+  if (!('clipboard' in navigator)) {
+    return;
+  }
+  document
+    .querySelector('main')
+    // ToDo: Use `:is(h2, h3, h4, h5, h6)[id]` once support is better.
+    .querySelectorAll('h2[id], h3[id], h4[id], h5[id], h6[id]')
+    .forEach((heading) => {
+      heading.addEventListener('click', (ev) => {
```

Comment:
```suggestion
      heading.addEventListener('click', (e) => {
```

I don't know if we have a hard rule on this but I think we tend to use `e` for event names. Although I do see one other instance of `(ev)`. But maybe let's go with `e` in this case.

Reply:
Done.